src/Schema.php

```diff
@@ -20,7 +20,7 @@ class Schema {
    * CORE_STANDARDS_HTTP_HEADERS.
    */
   const HTTP_RESPONSE_HEADERS = [
-    'Content-Security-Policy' => '"frame-ancestors"',
+    'Content-Security-Policy-Report-Only' => '"script-src \'unsafe-inline\' \'unsafe-eval\' http: https:"',
```

Comment:
I don't really understand why we need to add `script-src` now.  If none is specified, then the browser should not apply any restrictions for scripts.

Did you try this?
```suggestion
    'Content-Security-Policy' => '"frame-ancestors \'self\'"',
```

Reply:
Seems to work correctly on https://qa.bega.wohn.netz.rocks/wp-admin/post.php?post=445166&action=elementor

Reply:
yes seem to work here as well https://stage.gaco.netz.rocks/wp-admin/post.php?post=76422&action=elementor, although I changed it again for now to work only as a reporter and dig in a a bit more in a separate task as you mention